application/forms/Config/General/ApplicationConfigForm.php

```diff
@@ -70,22 +70,36 @@ public function createElements(array $formData)
             )
         );
 
-        $this->addElement(
-            'select',
-            'global_config_backend',
-            array(
-                'required'      => true,
-                'autosubmit'    => true,
-                'label'         => $this->translate('User Preference Storage Type'),
-                'multiOptions'  => array(
-                    'ini'   => $this->translate('File System (INI Files)'),
-                    'db'    => $this->translate('Database'),
-                    'none'  => $this->translate('Don\'t Store Preferences')
-                )
-            )
-        );
-
-        if (isset($formData['global_config_backend']) && $formData['global_config_backend'] === 'db') {
+        // we do not need this form for setup because we set the database there as default.
+        // this form is only displayed in configuration -> application if preferences backend type of ini is recognized
+        if (isset($formData['global_config_backend'])
+            && $formData['global_config_backend'] === 'ini'
```

Comment:
It's not too long for both conditions on the same line. Please remove that line break.